assets/src/util/date.js

```diff
@@ -12,13 +12,12 @@ const calculateWeekOffset = (startDateTime, targetDateTime) => {
   return Math.ceil((differenceInDays + 1) / 7)
 }
 
+// Get month day format of date string without timezone conversions
+// eg. timezone 2022-06-23T23:59:00 to 6/23
```

Comment:
I just wanted to be sure I understand the change here…

You changed it because using the `Date()` constructor uses the browser's time zone to adjust the given ISO 8601 timestamp, which we do not want.  Is that correct?

The comment is slightly confusing.